crypto/rand/drbg_lib.c

```diff
@@ -239,7 +311,7 @@ int RAND_DRBG_reseed(RAND_DRBG *drbg,
     if (!ctr_reseed(drbg, entropy, entropylen, adin, adinlen))
         goto end;
     drbg->state = DRBG_READY;
-    drbg->reseed_counter = 1;
+    drbg->reseed_counter = 0;
```

Comment:
Just in case anybody wonders: This fixes an off-by-one error introduced by me when removing the unorthodox exit condition [at the end of RAND_DRBG_generate()](https://github.com/openssl/openssl/commit/e0b625f9db00509af9004b7907d44b78f332754a#diff-9181ac017a6177a5f2619f65c9b7a346) (line 405/407) in commit e0b625f9db00509af9004b7907d44b78f332754a.